love/src/components/FlightTracker/FlightTracker.container.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { addGroup, removeGroup } from 'redux/actions/ws';
+import { getFlightTracker } from 'redux/selectors';
+import FlightTracker from './FlightTracker';
+import SubscriptionTableContainer from 'components/GeneralPurpose/SubscriptionTable/SubscriptionTable.container';
+
+export const schema = {
+  description: 'View of the Flight Tracker',
+  defaultSize: [30, 30],
+  props: {
+    title: {
+      type: 'string',
+      description: 'Name diplayed in the title bar (if visible)',
+      isPrivate: false,
+      default: 'Flight Tracker',
+    },
+    hasRawMode: {
+      type: 'boolean',
+      description: 'Whether the component has a raw mode version',
+      isPrivate: false,
+      default: false,
+    },
+  },
+};
+
+const FlightTrackerContainer = ({ isRaw, subscriptions, status, aircrafts }) => {
+  if (isRaw) {
+    return <SubscriptionTableContainer subscriptions={subscriptions} />;
+  }
+  return (
+    <FlightTracker
+      status={status}
+      aircrafts={aircrafts}
+    />
+  );
+};
+
+const mapStateToProps = (state) => {
+  const data = getFlightTracker(state);
+  return {
+    status: data.status,
+    aircrafts: data.aircrafts,
+  }
```

Comment:
Suggestion: this could be:
```javascript
const data = getFlightTracker(state);
return data;
``` 
And would yield the same result.

Reply:
Now try with return { ...data };